src/components/shared/forms/Checkbox.vue

```diff
@@ -33,12 +33,17 @@ export default {
             default: '',
         },
     },
-    data() {
-        return {
-            checker: this.checked,
-        }
-    },
+
     computed: {
+        checker: {
+            get() {
+                return this.checked
+            },
+            set(val) {
+                this.$emit('toggled', val)
```

Comment:
Can this also emit an 'input' event to keep it standard with the v-model pattern?